### ISSUE TITLE
pyroscope.tag_wrapper and other methods cause errors if pyroscope not configured

### ISSUE THREAD

**Issue by davidhao3300** [pyroscope-io/pyroscope-rs#70]
**Describe the bug you encountered:**

Repro:
```
import pyroscope

with pyroscope.tag_wrapper({"key": "value"}):
    print("hello world")
```

```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Os { code: 2, kind: NotFound, message: "No such file or directory" })', src/lib.rs:173:64
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
Aborted (core dumped)
```

**What did you expect to happen instead?**

Not have the program crash if pyroscope is not configured, right now I have to add a wrapper on top to no-op the tag_wrapper if pyroscope is not configured.

**How did you install `pyroscope-rs`?**

pip

**Comment by korniltsev** [comment]
fixed here [cd3c7d9](https://github.com/pyroscope-io/pyroscope-rs/commit/cd3c7d9a96ff8cef9300f761eff95b5fad3e8f0d)

**Comment by davidhao3300** [comment]
Thank you!
